include/openthread/thread.h

```diff
@@ -286,6 +286,18 @@ otError otThreadSetJoinerAdvertisement(otInstance *   aInstance,
                                        const uint8_t *aAdvData,
                                        uint8_t        aAdvDataLength);
 
+/**
+ * This method returns the Thread Joiner Advertisement.
+ *
+ * @param[in]   aInstance       A pointer to an OpenThread instance.
+ * @param[out]  aOui            The Vendor IEEE OUI value that will be included in the Joiner Advertisement.
+ * @param[out]  aAdvDataLength  The length of Advertisement data in bytes.
+ *
+ * @retval A pointer to the Advertisement data, or NULL if none is set.
+ *
+ */
+const uint8_t *otThreadGetJoinerAdvertisement(otInstance *aInstance, uint32_t *aOui, uint8_t *aAdvDataLength);
```

Comment:
Could you help explain what's the use case of these "Get" APIs and why do we need them?